test/libsolidity/semanticTests/array/reusing_memory.sol

```diff
@@ -28,4 +28,4 @@ contract Main {
 // f(uint256): 0x34 -> 0x46bddb1178e94d7f2892ff5f366840eb658911794f2c3a44c450aa2c505186c1
 // gas irOptimized: 117287
 // gas legacy: 127152
-// gas legacyOptimized: 113679
+// gas legacyOptimized: 114533
```

Comment:
Here is the diff: https://gist.github.com/hrkrshnn/0fe10f29134e1494297f0437fc788a48